sympy/functions/elementary/complexes.py

```diff
@@ -58,6 +58,8 @@ def eval(cls, arg):
             return arg
         elif arg.is_imaginary or (S.ImaginaryUnit*arg).is_real:
             return S.Zero
+        elif arg.is_Matrix:
+            return arg.as_real_imag()[0]
```

Comment:
@asmeurer Is this all right now ? 
I'm sorry that your comment got deleted.When I commited again there was a mix-up and had to use `git reset`